Zappy_IA_Report/DTC/Code/protocole.py

```
@@ -0,0 +1,210 @@
+    "Take",
+    "Set",
+    "Incantation",
+]
+
+
+def send_message(client, message) -> None:
+    formatted_message = message + "\n"
+    client["socket"].send((formatted_message).encode())
+
+
+def handle_Forward(client, response) -> None:
```

Ruff (0.11.9)

Missing type annotation for function argument `client`

(ANN001)